tests/test_boolean.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from dirty_equals import IsFalseLike
+
+
+@pytest.mark.parametrize(
+    'other, expected',
+    [
+        (False, IsFalseLike),
+        (True, ~IsFalseLike),
+        ([], IsFalseLike),
+        ([1], ~IsFalseLike),
+        ((), IsFalseLike),
+        ((1, 2), ~IsFalseLike),
+        ({}, IsFalseLike),
+        ({1: 'a'}, ~IsFalseLike),
+        (set(), IsFalseLike),
+        ({'a', 'b', 'c'}, ~IsFalseLike),
+        (None, IsFalseLike),
+        (object, ~IsFalseLike),
+        (0, IsFalseLike),
+        (1, ~IsFalseLike),
+        (0.0, IsFalseLike),
+        (1.0, ~IsFalseLike),
+        ('0', IsFalseLike(allow_strings=True)),
+        ('1', ~IsFalseLike(allow_strings=True)),
+        ('0.0', IsFalseLike(allow_strings=True)),
+        ('1.0', ~IsFalseLike(allow_strings=True)),
+        ('False', IsFalseLike(allow_strings=True)),
+        ('True', ~IsFalseLike(allow_strings=True)),
+    ],
+)
+class TestIsFalseLike:
+    def test_dirty_equals(self, other, expected):
+        assert other == expected
+
+    def test_dirty_not_equals(self, other, expected):
+        with pytest.raises(AssertionError):
+            assert other != expected
```

Comment:
this isn't achieving anything, you can remove it and make this a simple function test.